README.md

```diff
@@ -0,0 +1,65 @@
+# README
+
+## Table of contents
+
+1. [Context](#context)
+1. [Features](#features)
+1. [Developing](#developing)
+1. [Testing](#testing)
+
+
+## Context
+
+This is a ruby application that mimics the game hangman.  
+
+
```

Comment:
What do you think about adding a section showing how to play, which might begin like this:

```suggestion
## How to play

```sh
$ cd path/to/hangman
$ script/hangman
Welcome to Hangman!
Press Enter to begin
# insert pretend game interaction here
```
```

Reply:
👍 bb6d6fa7feff8b19424f7548f74363d1f349fdd2

Reply:
As a small note, it's common practice to have `./bin/[start-script].sh` as the directory.
Not a big deal, but you'll find in popular non-web Ruby applications that `bin` is usually the directory for that script 🙂